services/weather/docker-compose.yaml

```
@@ -0,0 +1,41 @@
+services:
+  redis:
+    image: "redis:8-alpine"
+    ports: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)